fearless/Common/DataProvider/Subscription/StakingLocalStorageSubscriber.swift

```diff
@@ -0,0 +1,501 @@
+import Foundation
+import RobinHood
+
+protocol StakingLocalStorageSubscriber where Self: AnyObject {
+    var stakingLocalSubscriptionFactory: StakingLocalSubscriptionFactoryProtocol { get }
+
+    var stakingLocalSubscriptionHandler: StakingLocalSubscriptionHandler { get }
+
+    func subscribeToMinNominatorBond(for chainId: ChainModel.Id) -> AnyDataProvider<DecodedBigUInt>?
+
+    func subscribeToCounterForNominators(for chainId: ChainModel.Id)
+        -> AnyDataProvider<DecodedU32>?
+
+    func subscribeMaxNominatorsCount(for chainId: ChainModel.Id)
+        -> AnyDataProvider<DecodedU32>?
+
+    func subscribeNomination(for accountId: AccountId, chainId: ChainModel.Id)
+        -> AnyDataProvider<DecodedNomination>?
+
+    func subscribeValidator(for accountId: AccountId, chainId: ChainModel.Id)
+        -> AnyDataProvider<DecodedValidator>?
+
+    func subscribeLedgerInfo(for accountId: AccountId, chainId: ChainModel.Id)
+        -> AnyDataProvider<DecodedLedgerInfo>?
+
+    func subscribeActiveEra(for chainId: ChainModel.Id) -> AnyDataProvider<DecodedActiveEra>?
+
+    func subscribeCurrentEra(for chainId: ChainModel.Id) -> AnyDataProvider<DecodedEraIndex>?
+
+    func subscribePayee(for accountId: AccountId, chainId: ChainModel.Id) -> AnyDataProvider<DecodedPayee>?
+
+    func subscribeTotalReward(
+        for address: AccountAddress,
+        api: ChainModel.ExternalApi,
+        assetPrecision: Int16
+    ) -> AnySingleValueProvider<TotalRewardItem>?
+
+    func subscribeStashItemProvider(
+        for address: AccountAddress
+    ) -> StreamableProvider<StashItem>?
```

Comment:
As far as I remember, we used to name functions `subscribeToSmth(...)`. I wonder what was the reason for changing that and why there is difference btw first two and the others?